### ISSUE TITLE
Ability to style `ul` treelist parent

### ISSUE THREAD

**Issue by sparanoid** [AnyRoad/react-json-view-lite#38]
With [this commit](https://github.com/AnyRoad/react-json-view-lite/commit/44f15ee274d60f68aba41e56eb9e427d8e6e84e7#diff-52594cd2543a6c522641a22b340f317545a500e0c0b2105d84367a192cef2bdbL123). the parent of the tree items changed from div to ul, which may introduce some styling issues.

In most frameworks, there are no default styles for `div`, but there are for `ul`. This change has caused styling issues in my case because we have default margin and padding for `ul`, but there's no way to add custom CSS like for [other elements](https://github.com/AnyRoad/react-json-view-lite#styleprops).

**Comment by AnyRoad** [comment]
Yes, I also realized it, will the css like below fix your case?

```
.basic-child-style > ul {
  margin: 0;
  padding: 0;
}
```

I will change this part to have better style support in the next version this week.

**Comment by sparanoid** [comment]
> will the css like below fix your case?

Yes this is my current workaround but it introduces extra CSS in a global file. A better style support for `ul` would be great.

**Comment by AnyRoad** [comment]
Hi, It's been a while, but I've finally released version 2.1.0 with the new style named `childFieldsContainer` 😄

**Comment by mindplay-dk** [comment]
it's still rendering `<ul>` with `<div>` children

<img width="788" height="127" alt="Image" src="https://github.com/user-attachments/assets/6ce15635-7037-4263-aa3a-130ff6733ae2" />

this isn't valid markup - a `<ul>` should only have `<li>` as direct children

browsers will usually render invalid markup without visibly breaking, but it can cause unexpected behavior with CSS, accessibility tools, and JS DOM queries etc.

the README says something about "implementation uses ul element div instead of the elemenent according to the w3.org example", but the example you point to correctly uses `<li>` items:

<img width="766" height="285" alt="Image" src="https://github.com/user-attachments/assets/a0d9c958-b8cd-4dee-ad9e-cfbfb4417fba" />

note that ARIA roles don't change HTML parsing rules, so I'm not sure where you got this idea 😊
